scipy/stats/_cdf_distance_1d.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+
+#pythran export _cdf_distance_1d(int, float[:], float[:])
+#pythran export _cdf_distance_1d(int, float[:], float[:], None, None)
+#pythran export _cdf_distance_1d(int, float[:], float[:], float[:], float[:])
```

Comment:
this should also work (and catch more cases):
```
#pythran export _cdf_distance_1d(int, float[:], float[:], float[:] or None, float[:] or None)
```

Reply:
Oh I forgot these two cases. Thank you very much !!